vertical-pod-autoscaler/pkg/updater/logic/updater.go

```diff
@@ -155,6 +165,17 @@ func (u *updater) RunOnce() {
 	timer.ObserveTotal()
 }
 
+func getRateLimiter(evictionRateLimit float64, evictionRateLimitBurst int) *rate.Limiter {
+	var evictionRateLimiter *rate.Limiter
+	if evictionRateLimit == -1 || evictionRateLimit == 0 {
+		evictionRateLimiter = rate.NewLimiter(rate.Inf, evictionRateLimitBurst)
+		klog.Warning("Rate limit disabled")
```

Comment:
I think this is Info level log, it's the default currently to have rate limit disabled.

Reply:
Indeed, it makes more sense 👍 I fixed it.